packages/shared/locales/en.json

```diff
@@ -1000,16 +1000,11 @@
         "version": "Version {version}",
         "unknown": "Unknown",
         "none": "None",
-        "days": "day(s)",
```

Comment:
This string appears to be unused and also assumes only 2 plural categories, so I removed it